module/VuFind/sql/mysql.sql

```diff
@@ -432,7 +433,8 @@ CREATE TABLE `login_token` (
   `expires` int NOT NULL,
   `last_session_id` varchar(255) NULL,
   PRIMARY KEY (`id`),
-  KEY `user_id` (`user_id`),
-  KEY `series` (`series`)
+  KEY `login_token_user_id_idx` (`user_id`),
+  KEY `login_token_series_idx` (`series`),
+  CONSTRAINT `login_token_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
-/*!40101 SET character_set_client = @saved_cs_client */;
+/*!40101 SET character_set_client = @saved_cs_client */;
```

Comment:
Looks like the training line break from this file got deleted. Harmless, but we try to keep trailing line breaks on files by default, so let's put it back:

```suggestion
/*!40101 SET character_set_client = @saved_cs_client */;

```